doc/rst/technotes/libraries.rst

```diff
@@ -343,8 +373,9 @@ files are generated, e.g.:
 
    export PYTHONPATH=lib/:$PYTHONPATH
```

Comment:
Is this step necessary by default now, or does the `__init__.py` file change that?

Reply:
The tests seem to explicitly unset the PYTHONPATH, so it seems like we should indicate it isn't necessary?

Reply:
If you create an init file then you can just import the entire directory as a package, which works like importing any other Python package. I left the old paragraphs because it's still a valid way of doing things if one doesn't want to make use of the init file. Should I just remove it?

Reply:
I think keeping it is the right call, since there are some strategies that still require it, but I'd be more explicit about when those situations occur